app/components/amplience/card-enhanced/index.jsx

```diff
@@ -119,13 +119,12 @@ const CardEnhanced = ({
     const [height, setHeight] = useState(400)
     const [width, setWidth] = useState(400)
     const [ratio, setRatio] = useState('1:1')
+    const [columns, setColumns] = useState(1)
+    const [rowss, setRows] = useState(1)
     const parentRef = useRef()
 
     const handleImageLoaded = () => {
         setImageLoading(false)
-        /*  if (onImageLoad instanceof Function) {
```

Comment:
There's a bunch of commented code in here now, but I don't know if it could come in handy later.